src/web/services/navigation.service.spec.ts

```diff
@@ -14,4 +14,9 @@ describe('NavigationService', () => {
     const service: NavigationService = TestBed.get(NavigationService);
     expect(service).toBeTruthy();
   });
+
+  it('should return a encoded param string upon calling encodeParams', () => {
```

Comment:
Small grammatical error: should be `an encoded param string` rather than `a encoded param string`.